### ISSUE TITLE
Feature Request: Send GIFs || Send video as gif from URL

### ISSUE THREAD

**Issue by AlenSaito1** [open-wa/wa-automate-nodejs#1221]
It'd be great if there was a way to send GIF files or Video as gif from URL (Non-giphy) instead of downloading the GIFs and converting to video

**Comment by smashah** [comment]
@AlenSaito1 if this is implemented you will need to use [`client.sendVideoAsGif `](https://docs.openwa.dev/classes/client.html#sendvideoasgif) and only use mp4 URLs

**Comment by AlenSaito1** [comment]
Perfect, Thank you

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [3.1.13](https://github.com/open-wa/wa-automate-nodejs/compare/3.1.12...3.1.13) (2021-01-13)

- build(deps): bump sharp from 0.26.3 to 0.27.0 [`#1195`](https://github.com/open-wa/wa-automate-nodejs/pull/1195)
- build(deps): bump boxen from 4.2.0 to 5.0.0 [`#1203`](https://github.com/open-wa/wa-automate-nodejs/pull/1203)
- build(deps): bump swagger-ui-express from 4.1.5 to 4.1.6 [`#1194`](https://github.com/open-wa/wa-automate-nodejs/pull/1194)
- build(deps): bump ts-json-schema-generator from 0.81.0 to 0.82.0 [`#1197`](https://github.com/open-wa/wa-automate-nodejs/pull/1197)
- build(deps): bump cfonts from 2.9.0 to 2.9.1 [`#1201`](https://github.com/open-wa/wa-automate-nodejs/pull/1201)
- build(deps-dev): bump ava from 3.14.0 to 3.15.0 [`#1216`](https://github.com/open-wa/wa-automate-nodejs/pull/1216)
- build(deps-dev): bump @types/node from 14.14.14 to 14.14.20 [`#1219`](https://github.com/open-wa/wa-automate-nodejs/pull/1219)
- build(deps): bump @types/sharp from 0.26.1 to 0.27.1 [`#1223`](https://github.com/open-wa/wa-automate-nodejs/pull/1223)
- build(deps): bump meow from 8.0.0 to 9.0.0 [`#1225`](https://github.com/open-wa/wa-automate-nodejs/pull/1225)
- build(deps-dev): bump typedoc from 0.19.2 to 0.20.14 [`#1229`](https://github.com/open-wa/wa-automate-nodejs/pull/1229)
- Release 3.1.13 [`e91ae2d`](https://github.com/open-wa/wa-automate-nodejs/commit/e91ae2d80a458fe6cd84301bdd940e3eeeb16fcc)
- fix typedoc dependency [`46f1a74`](https://github.com/open-wa/wa-automate-nodejs/commit/46f1a742c729be4c810442bd66b67235854fe74c)
- Feature: Can now send a mp4 as a GIF using a URL #1221 [`fcb7a3b`](https://github.com/open-wa/wa-automate-nodejs/commit/fcb7a3b5690372a8d35b912c238da833cd1f02d9)
- Feature: Implemented `getGroupInfo` #1230 [`888382b`](https://github.com/open-wa/wa-automate-nodejs/commit/888382b337b14b67baf1d7dd2cb7361576fb1109)

**Comment by smashah** [comment]
@AlenSaito1 in the latest version you can send a video as gif using URL. Example:

```javascript
      const mp4AsGifViaUrl = await client.sendVideoAsGif(message.from,'https://file-examples-com.github.io/uploads/2017/04/file_example_MP4_480_1_5MG.mp4','test.gif','wow cool gif')
```

Thanks
